mvpa2/generators/resampling.py

```diff
@@ -203,3 +205,92 @@ def generate(self, ds):
     def __str__(self):
         return _str(self, str(self._amount), n=self._attr, count=self.count,
                     apply_selection=self._apply_selection)
+
+
+class LogExclusions(Node):
+    """Log excluded entries
+
+    Given a dataset with a boolean sample or feature attribute, return a
+    dataset with only those samples/features marked `True`.
+    """
+    def __init__(self, fname, append=True, space='balanced_set', **kwargs):
+        """
+        Parameters
+        ----------
+        space : str
+          Name of the selection marker attribute in the input dataset that
+          indicates the desired subset.
+        """
+        Node.__init__(self, space=space, **kwargs)
```

Comment:
best is to use `super` while invoking super class's constructor... we do use multiple inheritance at places